sites/next.skeleton.dev/src/content/docs/guides/figma.mdx

```diff
@@ -0,0 +1,27 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+title: Figma UI kit
+description: Welcome to the Skeleton Figma Design System Tutorials!
+order: 60
+---
+
+export const components = componentSet;
+
+import FigmaBuyNow from '@components/docs/FigmaBuyNow.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
+
+import { Image } from 'astro:assets';
+import ImgHero from '@images/get-started/figma/hero.webp';
+import ImgComponents from '@images/get-started/figma/components.webp';
+import ImgPlugin from '@images/get-started/figma/plugin.webp';
+import ImgTutorial from '@images/get-started/figma/tutorials.webp';
+import ImgKit from '@images/get-started/figma/ui-kit.webp';
+import { Clock3 } from 'lucide-react';
+import { Shapes } from 'lucide-react';
+import { SwatchBook } from 'lucide-react';
+
+Explore step-by-step guides to unlock the full potential of Skeleton in your design workflow. Whether you're setting up for the first time or mastering advanced features, these tutorials will guide you every step of the way.
```

Comment:
Feel free to make this the larger prominent text as seen on most other doc pages.

Example:
https://skeleton-next-krbzc29ja-skeleton-labs.vercel.app/docs/get-started/fundamentals